### ISSUE TITLE
Improve display of loaded files

### ISSUE THREAD

**Issue by mattatgit** [vizorvr/vizor#1163]
What i have in mind here applies first to the Texture loader patches, because going back to the Solar System graph to optimise it, but i have absolutely no information about how i've set it up. But it also applies to the model loaders.

To be specific:

I don't know which texture files are loaded onto which models, except by visually inspecting the rendered planet. For example, I have several different texture files for Mars' surface. Free ones, better ones i bought, with different names and in various resolutions. 
* don't know what resolution i'm using
* don't know the file name
* don't know how heavy (KB/ MB) the files are.

Two different possible approaches, as seen in these two examples below
![patch information](https://cloud.githubusercontent.com/assets/6858525/12738358/f2c2c62e-c9b3-11e5-9012-b68b912497d0.png)

LHS – file name, size & resolution displaying on the tile, overlay over the image.
RHS – file name is read & inserted into the Patch name. This might be more work, potentially lead to some wide patches due to long names, but probably makes more sense & is more visible. File size & pixel dimensions overlaid on the image.

Thoughts ? @fthr @kschzt @gmarinov @kkananen

**Comment by fthr** [comment]
Yeah, agree that this is necessary, I've had similar problems. I prefer the left hand side version.

As for 3D objects, the information I usually look for would be polygon count and file size.

I also think the assets should be able to have a couple more pieces of metadata:
-Original uploader if not yourself
-Tags for easier search
-Description written by uploader (this could also be used for attribution for assets that are not yours).

That metadata shouldn't display in the plugin itself as in the designs above, but should be available in the asset browser window.

**Comment by mattatgit** [comment]
Yeah thinking further about this, while having the file name load into the patch name would be handy, it would be lost if patch name is changed. So yeah, i'll do another version based on the left hand side one. 

This seems to be becoming more important, as for example you try to modify work by other people whose original files you don't have (ie me remixing Antti's 360 carousel), but you don't know the size of the images, or the size of the plane in px etc. Just means a bunch of tedious guesswork. @fthr

**Comment by mattatgit** [comment]
We should probably implement the left hand side version of the enhancement shown above. It might not have all the metadata @fthr is after but it would help a lot even with this level of data.
